README.md

```diff
@@ -34,35 +58,79 @@ $ ansible-playbook -i localhost, --connection=local \
 [![asciicast](https://asciinema.org/a/161278.png)](https://asciinema.org/a/161278)
 
 
-## Deploy new cluster + KubeVirt
+## Deploy Kubernetes or OpenShift and KubeVirt
 
+In order to deploy new cluster with KubeVirt on top of it, you need to perform 2 steps
 
-### Requirements
-Install depending roles, and export `ANSIBLE_ROLES_PATH`
-```
-$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
-$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
-```
-For OpenShift deployment clone [**OpenShift Ansible project**](https://github.com/openshift/openshift-ansible)
-```
-$ git clone https://github.com/openshift/openshift-ansible
-```
+1. Run one of following playbooks to deploy new cluster
+
+  * [deploy-kubernetes.yml](./deploy-kubernetes.yml) for Kubernetes cluster
+  * [deploy-openshift.yml](./deploy-openshift.yml) for OpenShift cluster
+
+2. Run [install-kubevirt-on-openshift.yml](./install-kubevirt-on-openshift.yml) playbook to install KubeVirt
+
+  Follow [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-openshift-cluster) section
 
 ### Kubernetes
-Preparing Kubernetes cluster and deploy KubeVirt on it.
+
 - Add your master and nodes to `inventory` file.
-- Run ansible playbook `# ansible-playbook -i inventory deploy-kubernetes.yml`.
+
+```bash
+$ # 1) deploy Kubernetes
+$ ansible-playbook -i inventory deploy-kubernetes.yml
+$ # 2) install KubeVirt
+$ ansible-playbook ... install-kubevirt-on-openshift.yml
```

Comment:
This doesn't make sense. We don't have playbook which installs KubeVirt on Kubernetes